hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/common/Checksum.java

```diff
@@ -154,11 +158,6 @@ public ChecksumData computeChecksum(List<ByteString> byteStrings)
 
   public ChecksumData computeChecksum(ChunkBuffer data)
       throws OzoneChecksumException {
-    if (checksumType == ChecksumType.NONE) {
-      // Since type is set to NONE, we do not need to compute the checksums
-      return new ChecksumData(checksumType, bytesPerChecksum);
-    }
-
```

Comment:
On a second thought, we should not remove this if-statement since some other code paths depend on it.  Let's revert the commit.

Reply:
Yes, I hadn't noticed that this method was used by other codes.  
I will submit a new PR. We only need to add judgments to computeChecksum(ByteBuffer Data) to avoid redundant ByteBuffer conversions in this method